### PR TITLE
Generate migrations with declared strict types

### PR DESCRIPTION
**Pull request by radekhubner**

### CODE REVIEW

File: lib/Doctrine/DBAL/Migrations/Tools/Console/Command/GenerateCommand.php [hunk 1]
```diff
@@ -38,7 +38,7 @@ class GenerateCommand extends AbstractCommand
 {
 
     private static $_template =
-            '<?php
+            '<?php declare(strict_types = 1);
```
Comment (alcaeus) [inline lib/Doctrine/DBAL/Migrations/Tools/Console/Command/GenerateCommand.php]: Master still allows for PHP 5, which could cause errors for some people. I'd suggest dropping 5.x support from master first, then adding this in.

Comment (radekhubner) [inline lib/Doctrine/DBAL/Migrations/Tools/Console/Command/GenerateCommand.php]: Thank you

Comment (fidraj) [reply lib/Doctrine/DBAL/Migrations/Tools/Console/Command/GenerateCommand.php]: It's been dropped https://github.com/doctrine/migrations/blob/20dc1f98387ae33bba0f6a621815750b518d98f8/composer.json#L14

**Comment by mikeSimonson** [comment]
@radekhubner Thanks

**Pull request merged**